indico/MaKaC/services/implementation/category.py

```diff
@@ -159,7 +159,8 @@ def _getAnswer( self ):
         skip = max(0, self._lastIdx - 100)
         pastEvents = {}
         num = 0
-        for event in islice(index.itervalues(), skip, self._lastIdx):
+        events = [e for e in index.itervalues() if not e.as_event.is_deleted]
```

Comment:
no, a legacy event object doesn't know that it's deleted - but `ConferenceHolder().hasKey(event_id)` will return 0 if it's not in there anymore